haros/haros.py

```diff
@@ -317,10 +319,14 @@ def main(argv = None, source_runner = False):
                             level = logging.DEBUG)
     else:
         logging.basicConfig(level = logging.WARNING)
+
     try:
+        if args.dir:
+            os.chdir(args.dir)
```

Comment:
Windows is not really a target for ROS1, but for ROS2 it is, so if @git-afsantos sees a use for this tool beyond ROS1, then it might make sense to keep it in mind.

I don't know what other cross-platform issues there might be, but might be nice not to introduce any new ones.